Emit nop1,nop5 instructions for x86_64 usdt probe

olsajiri opened the pull request:
Now that we have no5 optimization support in kernel, let's emit
nop,nop5 for usdt probe. We leave it up to the library to use
desirable nop instruction.

anakryiko commented:
@olsajiri  can you please check what's wrong with x86-64 tests and fix them up as necessary?

olsajiri committed:
lib: Emit nop,no5 instructions combo for x86_64 arch

Now that we have no5 optimization support in kernel, let's emit
nop,nop5 for usdt probe. We leave it up to the library to use
desirable nop instruction.

Signed-off-by: Jiri Olsa <jolsa@kernel.org>
